css/style.css

```diff
@@ -833,3 +834,8 @@ video {
 .detailCallInfoContainer .password-input {
 	width: 100%;
 }
+
+.detailCallInfoContainer #link-checkbox+label {
```

Comment:
What is `+` in CSS rules?

Reply:
It refers to the element directly after the other. In this case the label directly after the input#link-checkbox (which itself does not have a class or ID). :)

Reply:
Ah, interesting